_layouts/documentation.html

```diff
@@ -26,7 +26,7 @@
 
           <div class="usa-display">{{ page.title }}</div>
 
-          {% if page.lead %}
+          {% if page.lead | markdownify %}
```

Comment:
There's another reference below on line 46 we should probably update as well, at least for consistency?